test_data/rules/experiments/nonfunction-type-aliases/rules/always_specify_types.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2015, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `dart test -N always_specify_types`
+
+import 'package:meta/meta.dart';
+
+typedef JsonMap = Map<String, dynamic>; //OK
+typedef StringList = List<JsonMap>; //OK
+typedef RawList = List; //LINT
+typedef StringMap<V> = Map<String, V>; //OK
```

Comment:
^^^ are the new test cases.

Reply:
I'd like to see a slightly more complicated test added:
```
typedef MapList = List<StringMap>; // LINT
```
This should create a lint because `StringMap` is generic but doesn't specify type arguments, but I suspect that the condition you added will cause it to not get caught.

Reply:
Should now be fixed. 🤞